documentation (#32)
infinum/Locker: README.md

<string>** Add your Face ID Usage Description **</string>
```

##### 1. Save Your data with `setSecret: forUniqueIdentifier: completed:` method. 
For `uniqueIdentifier` pass `String` value You will later use to fetch Your data.

> The `completed` is a closure which you can pass to handle the error if it occurs.

Reviewer: ```suggestion
The `completed` is a closure that is called upon finished secret storage. If the error occurs upon storing, info will be passed through the completion block.
```

```swift
// Swift
let biometrySettingsChanged = Locker.biometricsSettingsDidChange
let usingBiometry = Locker.shouldUseAuthenticationWithBiometrics(for: "kUniqueIdentifier")
if biometrySettingsChanged && usingBiometry {
// handle case when settings are changed and biometry should be used
}
```

##### 7. There are `supportedBiometricsAuthentication` and `configuredBiometricsAuthentication` computed properties which return `BiometricsType` enum (`BiometricsTypeNone`, `BiometricsTypeTouchID`, `BiometricsTypeFaceID`).
`supportedBiometricsAuthentication` checks if the device has support for some Biometric type.
`configuredBiometricsAuthentication` checks if the device has support for some Biometrics type and if that Biometric is enabled in device settings.

> #### 8. There is a local JSON file which contains every iPhone and iPad model which has FaceID or TouchID. That way we can check if the user's device can use FaceID or TouchID. If you want to allow the JSON file to sync itself with an API if the user's device isn't contained in the list, you can set `enableDeviceListSync` to be true.
> `enableDeviceListSync` checks if the user's device is on the JSON file biometry supported device list. If the device is not on the list, it syncs the list with a list from the API and writes it down to the local JSON file.

Reviewer: ```suggestion
#### 8. There is a local JSON file that contains every iPhone and iPad model which has FaceID or TouchID. That way we can check if the user's device can use FaceID or TouchID. If you want to allow the JSON file to sync itself with a server, you can set `enableDeviceListSync` to `true`.
`enableDeviceListSync` when enabled, if the device is not present on the local list, it syncs the list with a list from the server and writes it down to the local JSON file.
```
